app/controllers/items_controller.rb

```diff
@@ -15,6 +10,11 @@ def create
     end
   end
 
+  def index
+    @item = Item.new
+    @category = Category.new
```

Comment:
It's generally a good practice to only set up one variable in the controller. Since it looks like you're not using this one, I'd recommend removing it.